libs/sdk-ui-filters/src/AttributeFilter/AttributeFilterBase.tsx

```diff
@@ -10,12 +10,12 @@ import { AttributeFilterProviders } from "./AttributeFilterProviders";
  * @internal
  */
 export const AttributeFilterBase: React.FC<IAttributeFilterBaseProps> = (props) => {
-    const backend = useBackend(props.backend);
+    const backend = useBackendStrict(props.backend).withTelemetry("AttributeFilter", props);
```

Comment:
This is dangerous, we should memoize this aggressively.

Reply:
withTelemetry returns a new instance everytime, so the backend will invalidate everything in its path.